poker/.eslintrc

```diff
@@ -11,7 +16,13 @@
   },
   "rules": {
     "comma-dangle": "off",
+    "no-console": 0,
```

Comment:
bad rule

Reply:
I agree, I will remove it after the release